### PR TITLE
Refactor/packages update

### LINKED ISSUE #10: Integrate the new versions of generators
Once v0.7 is live, we need to switch from installing the old `teleport-code-generators` npm package to the new individual packages.

### PR DESCRIPTION
**Pull request by xavxyz**
Context
---
Fixes #10 as version 0.7.0 of generators got published! 🚀 

Proposed solution
---
- Use individual packages instead of the main big one
- Manage available plugins with a list (InlineStyles, JSS, StyledJSX, CSSModules) to get in a glance what are the available flavors.
- Remove `package-lock.json` as Yarn is used on all TeleportHQ OSS projects

Tests
---
- [ ] Install new deps, `yarn`
- [ ] Run the playground `yarn dev`
- [ ] Select a component featuring styles in its UIDL, like `expandable-area`
- Code is generated as expected for React flavors: 
  - [ ] InlineStyles
  - [ ] JSS
  - [ ] StyledJSX
  - [ ] CSSModules
- [ ] Code is generated as expected for Vue

### CODE REVIEW

File: components/CodeScreen/index.tsx [hunk 1]
```diff
@@ -27,22 +24,17 @@ import { Tabs } from '../Tabs'
...
-  variation: ReactComponentStylingFlavors.CSSModules,
-  customMapping: nextMapping,
-})
+
+const reactStylesPlugins = ['InlineStyles', 'JSS', 'StyledJSX', 'CSSModules']
+const reactGenerators: Record<string, ComponentGenerator> = reactStylesPlugins.reduce(
+  (table, plugin) => ({
+    ...table,
+    [plugin]: createReactComponentGenerator(plugin, {
+      mapping: nextMapping,
```
Comment (alexnm) [inline components/CodeScreen/index.tsx]: is there any reason you added the nextMapping for the generators here?

Comment (xavxyz) [reply components/CodeScreen/index.tsx]: I was keeping consistent from the previous code, but I wasn't sure of its use! If there is no plan for export to Next, maybe we don't need them at all!

Comment (alexnm) [reply components/CodeScreen/index.tsx]: no, you can remove it. I think the older version was based on some legacy use case.

Comment (xavxyz) [reply components/CodeScreen/index.tsx]: 👌, removed!

**Pull request merged**